stub-connector/src/main/java/uk/gov/ida/notification/stubconnector/ConnectorNodeCredentialConfiguration.java

```diff
@@ -44,7 +45,7 @@ public ConnectorNodeCredentialConfiguration(
         if (samlSigningCredential.getPublicKey() instanceof ECPublicKey) {
             this.algorithm = SignatureConstants.ALGO_ID_SIGNATURE_ECDSA_SHA256;
         } else {
-            this.algorithm = SignatureConstants.ALGO_ID_SIGNATURE_RSA_SHA256;
+            this.algorithm = XMLSignature.ALGO_ID_SIGNATURE_RSA_SHA256_MGF1;
```

Comment:
This is interesting - and I assume it's simply because `RSA_SHA256` isn't compliant but `RSA_SHA256_MGF1` is?